pkg/flags/common.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2024 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flags
+
+import (
+	"os"
+
+	"github.com/abcxyz/pkg/cli"
+)
+
+type CommonFlags struct {
+	FlagDir string
+}
+
+func (c *CommonFlags) Register(set *cli.FlagSet) {
+	f := set.NewSection("COMMON OPTIONS")
+
+	cwd, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
```

Comment:
I'm not sure if there's a better way to handle this err.

Reply:
I'm not sure that would help as the `Flags()` method does not allow returning an error which is where we would be calling the `Register` method.

https://github.com/abcxyz/guardian/blob/main/pkg/commands/plan/plan.go#L95